### ISSUE TITLE
set attachedView to keyWindow causing black screen

### ISSUE THREAD

**Issue by litt1e-p** [adad184/MMPopupView#25]
```swift
MMAlertView *alertView = [[MMAlertView alloc] initWithTitle:kATitle detail:kATip items:items];
alertView.attachedView = [UIApplication sharedApplication].keyWindow;
```
alertView can show normally, but it turns black screen after click every item.

environment: Xcode7.3

**Comment by adad184** [comment]
thanks, problem fixed now, plz use the latest version: 1.7.1

**Comment by litt1e-p** [comment]
thx

2016年3月29日星期二，ralph li <notifications@github.com> 写道：
